src/http_module.cpp

```diff
@@ -590,7 +608,7 @@ void exitWorkerProcess(ngx_cycle_t* cycle)
 
 char* setExporter(ngx_conf_t* cf, ngx_command_t* cmd, void* conf)
 {
-    auto mcf = (MainConf*)conf;
+    auto mcf = getMainConf(cf);
```

Comment:
Casting `conf` might be safer/more in-line with the Nginx codebase, especially if we decide to support `"otel_exporter"` in server/location contexts where `cf` and `conf` don't match.

```
 auto mcf = static_cast<MainConf*>((MainConfBase*)conf);
```